modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -6,16 +6,16 @@
 [id="rosa-hcp-classic-comparison_{context}"]
 = Comparing ROSA Classic and hosted control planes for ROSA
```

Comment:
I have a couple of suggestions/questions for this title. First, I am not sure if "Classic" should be capitalized or not. Second, while "hosted control planes" is technically accurate given the list of terms you gave me, you may want to use "ROSA with hosted control planes" like you do in the rest of the document. Finally, this title first lists the classic ROSA and then the hosted control planes version of ROSA, but the table that follows reverses that order: the first column is the ROSA with hosted control planes info and the second column is the classic ROSA info. Not a huge deal, but it doesn't follow parallelism. None of these are major issues, so feel free to take them or leave them.